prosper/common/prosper_logging.py

```diff
@@ -116,19 +117,47 @@ def close_handles(self):
         for handle in self.log_handlers:
             try:
                 handle.close()
-            except Exception:   #pragma: no cover
+            except Exception:
                 warnings.warn(
                     'WARNING: unable to close logging handle',
                     RuntimeWarning
                 )
-                #TODO:testable?
                 pass #do not crash if can't close handle
 
+    def _configure_common(self, prefix, fallback_level, fallback_format, handler_name, handler):
+        """commom configuration code
+        
+        Args:
+            prefix (str): A prefix for the `log_level` and `log_format` keys to use with the config. #FIXME: Hacky, add separate sections for each logger config?
+            fallback_level (str): Fallback/minimum log level, for if config does not have one.
+            fallback_format (str): Fallback format for if it's not in the config.
+            handler_name (str): Handler used in debug messages.
+            handler (str): The handler to configure and use.
+
+        """
+        ## Retrieve settings from config ##
+        log_level = self.config.get_option('LOGGING', prefix + 'log_level', None, fallback_level)
+        if logging.getLevelName(log_level) < logging.getLevelName(fallback_level):
+            log_level = fallback_level #  fallback is also the minimum
+
+        log_format_name = self.config.get_option('LOGGING', prefix + 'log_format', None, None)
+        log_format = ReportingFormats[log_format_name].value if log_format_name else fallback_format
+
+        ## Attach handlers/formatter ##
+        formatter = logging.Formatter(log_format)
+        handler.setFormatter(formatter)
+        self.logger.setLevel(log_level)
```

Comment:
```python
handler.setLevel(log_level)
```
needs to set the level on handler (or else master logger will only log to last-handler's levels.